docs/_includes/Programs.md

```diff
@@ -117,40 +116,33 @@ The `static` modifier is used for class members that that
 are associated with the class as a whole rather than with
 an instance of the class.
 
-The `protected` modifier is used to control the visibility of the
-body of functions.
-
-The `extern` modifier is used to alter the CompileName of
-entities. The CompileName is the name for the entity
-when translating to Boogie or a target programming language.
-
 The following table shows modifiers that are available
 for each of the kinds of declaration. In the table
 we use already-ghost (already-non-ghost) to denote that the item is not
 allowed to have the ghost modifier because it is already
 implicitly ghost (non-ghost).
 
 
- Declaration              | allowed modifiers                     
+ Declaration              | allowed modifiers
 --------------------------|---------------------------------------
- module                   | abstract                              
- class                    | extern                                
- trait                    | -                                     
- datatype or codatatype   | -                                     
- field                    | ghost                                 
- newtype                  | -                                     
- synonym types            | -                                     
- iterators                | -                                     
- method                   | ghost static extern                   
- lemma, colemma, comethod | already-ghost static protected        
- inductive lemma          | already-ghost static                  
- constructor              | -                                     
- function (non-method)    | already-ghost static protected        
- function method          | already-non-ghost static protected extern 
- predicate (non-method)   | already-ghost static protected        
- predicate method         | already-non-ghost static protected extern 
- inductive predicate      | already-ghost static protected        
- copredicate              | already-ghost static protected        
+ module                   | abstract
+ class                    | -
+ trait                    | -
+ datatype or codatatype   | -
+ field                    | ghost
+ newtype                  | -
+ synonym types            | -
+ iterators                | -
+ method                   | ghost static
+ lemma, colemma, comethod | already-ghost static
+ inductive lemma          | already-ghost static
+ constructor              | -
+ function (non-method)    | already-ghost static
+ function method          | already-non-ghost static
```

Comment:
Was it intentional to omit extern in various places on this list?

Reply:
Yes, but perhaps that was foolish. As I edited the table, I saw these mentions of the `extern` keyword, which is no longer supported, so I figured I'd just remove them while I was at it.